en/deploy-on-azure-aks.md

```diff
@@ -0,0 +1,552 @@
+    --cluster-name ${clusterName} \
+    --resource-group ${resourceGroup} \
+    --node-vm-size ${nodeType} \
+    --zones 1 2 3 \
+    --aks-custom-headers EnableAzureDiskFileCSIDriver=true \
+    --node-count 3 \
+    --labels dedicated=ticdc \
+    --node-taints dedicated=ticdc:NoSchedule
+```
+
+### Configure and deploy
+
++ To deploy TiFlash, configure `spec.tiflash` in `tidb-cluster.yaml`:
+
+    ```yaml
+    spec:
+      ...
+      tiflash:
+        baseImage: pingcap/tiflash
+        replicas: 1
+        storageClaims:
+        - resources:
+            requests:
+              storage: 100Gi
+        tolerations:
+        - effect: NoSchedule
+          key: dedicated
+          operator: Equal
+          value: tiflash
+    ```
+
+    For other parameters, refer to [Configure a TiDB Cluster](configure-a-tidb-cluster.md).
+
+    > **Warning:**
+    >
+    > TiDB Operator automatically mount PVs **in the order of the configuration** in the `storageClaims` list. Therefore, if you need to add disks for TiFlash, make sure that you add the disks **only to the end of the original configuration** in the list. In addition, you must **not** alter the order of the original configuration.
+
++ To deploy TiCDC, configure `spec.ticdc` in `tidb-cluster.yaml`:
+
+    ```yaml
+    spec:
+      ...
+      ticdc:
+        baseImage: pingcap/ticdc
+        replicas: 1
+        tolerations:
+        - effect: NoSchedule
+          key: dedicated
+          operator: Equal
+          value: ticdc
+    ```
+
+    Modify `replicas` according to your needs.
+
+Finally, execute `kubectl -n tidb-cluster apply -f tidb-cluster.yaml` to update the TiDB cluster configuration.
+
+For detailed CR configuration, refer to [API references](https://github.com/pingcap/tidb-operator/blob/master/docs/api-references/docs.md) and [Configure a TiDB Cluster](configure-a-tidb-cluster.md).
+
+## Deploy TiDB Enterprise Edition
+
+To deploy TiDB/PD/TiKV/TiFlash/TiCDC Enterprise Edition, configure `spec.[tidb|pd|tikv|tiflash|ticdc].baseImage` in `tidb-cluster.yaml` as the enterprise image. The enterprise image format is `pingcap/[tidb|pd|tikv|tiflash|ticdc]-enterprise`.
+
+For example:
+
+```yaml
+spec:
+  ...
+  pd:
+    baseImage: pingcap/pd-enterprise
+  ...
+  tikv:
+    baseImage: pingcap/tikv-enterprise
+```
+
+## Use other Disk volume types
+
+Azure disks supports multiple volume types. If you need low latency and high throughput, you can choose the `UltraSSD` type. The steps are as follows:
+
+1. [Enable Ultra disks on an existing cluster](https://docs.microsoft.com/en-us/azure/aks/use-ultra-disks#enable-ultra-disks-on-an-existing-cluster) and create a storage class for `UltraSSD`:
+
+    ```yaml
+    apiVersion: storage.k8s.io/v1
+    kind: StorageClass
+    metadata:
+      name: ultra
+    provisioner: disk.csi.azure.com
+    parameters:
+      skuname: UltraSSD_LRS  # alias: storageaccounttype, available values: Standard_LRS, Premium_LRS, StandardSSD_LRS, UltraSSD_LRS
```

Comment:
We should also set the mountOptions here? Refer to #1427

Reply:
added